README.md

```diff
@@ -42,16 +42,27 @@ with:
   all versioned paths: True
 ```
 
+> **Note**: This will also run `optimade-validator` for the unversioned base URL.
+
+By default, the validator follows the [OPTIMADE specification](https://github.com/Materials-Consortia/OPTIMADE/blob/master/optimade.rst).
+This means it will always check the mandatory base URLs:
+
+- Unversioned base URL (`/`)
+- Major-versioned base URL (`/vMAJOR`, e.g. `/v1`)
+
+The major version number will be determined based on the validator version used, i.e., the supported OPTIMADE API version in the [OPTIMADE Python tools repository](https://github.com/Materials-Consortia/optimade-python-tools/blob/master/optimade/__init__.py#L2).
+This can be chosen using the input `validator_version`.
+
 ## Inputs
 
 | Input | Description | Usage | Default | Action version |
 | :---: |    :---     | :---: |  :---:  |      :---:     |
-| `all_versioned_paths` | Whether to test all possible versioned base URLs:<br><br>/vMAJOR<br>/vMAJOR.MINOR<br>/vMAJOR.MINOR.PATCH<br><br>If this is `'true'`, the input `'path'` MUST exempt the version part (e.g., `'/optimade'` instead of `'/optimade/v1'`).<br>If this is `'false'`, the input `'path'` MUST include the version part (e.g., `'/optimade/v1'` instead of `'/optimade'`) | Optional | `false` | `v1+`
+| `all_versioned_paths` | Whether to test all possible versioned base URLs:<br><br>/vMAJOR<br>/vMAJOR.MINOR<br>/vMAJOR.MINOR.PATCH<br><br>The latter two being optional base URLs according to the specification. | Optional | `false` | `v1+`
 | `as_type` | Validate the request URL with the provided type, rather than scanning the entire implementation<br>Example values: 'structures', 'reference'. For a full list of values see `optimade-python-tools`. | Optional | - | `v1+`
 | `domain` | Domain for the OPTIMADE URL (defaults to the GitHub Actions runner host) | Optional | `gh_actions_host` | `v1+`
 | `fail_fast` | Whether or not to exit and return a non-zero error code on first failure. | Optional | `false` | `v2+`
 | `index` | Whether or not this is an index meta-database | Optional | `false` | `v1+`
-| `path` | Path for the OPTIMADE (versioned) base URL - MUST start with `/`<br>_Note_: If `all versioned paths` is `true`, this MUST be un-versioned, e.g., `/optimade` or `/`, otherwise it MUST be versioned, e.g., the default value | Optional | `/v1` | `v1+`
+| `path` | Path for the OPTIMADE (versioned) base URL - MUST start with `/`<br>The path MUST NOT include the versioned part of the base URL. Rather, it MUST point to the unversioned base URL of your OPTIMADE implementation. | Optional | `/` | `v1+`
```

Comment:
Could you clarify this a bit more? I know what you mean, but now the example value has changed to just `"/"` it might be unclear:

```suggestion
| `path` | Path to append to the domain to reach the OPTIMADE unversioned base URL - MUST start with `/`<br>The path MUST NOT include the versioned part of the base URL. Rather, it MUST point to the unversioned base URL of your OPTIMADE implementation` | Optional | `/` | `v1+`
```

Does this setting still make sense now or could all this info be bundled into the domain?

Reply:
Since the domain does not include the path, it cannot be added to the domain. Or do you mean to add `/` to the domain if not already present? That might be a great idea.

I put in all these `MUST BE /` though, to avoid having to do a million checks for prep- or appended `/`.